src/test/codegen/function-arguments.rs

```diff
@@ -11,7 +11,7 @@ pub struct UnsafeInner {
   _field: std::cell::UnsafeCell<i16>,
 }
 
-// CHECK: zeroext i1 @boolean(i1 zeroext %x)
+// CHECK: noundef zeroext i1 @boolean(i1 noundef zeroext %x)
```

Comment:
It would probably be good to test that `MaybeUninit<bool>` does not set noundef.